src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
+    protected String color;
```

Comment:
The 'color' field is declared as 'protected'. It's better to encapsulate it by declaring it private and providing a getter method to access its value if needed.